HerPortal/Views/Home/ReferralFiles.cshtml

```diff
@@ -0,0 +1,45 @@
+﻿@model HerPortal.Models.HomepageViewModel
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@{
+    ViewBag.Title = "Home Energy Retrofit Portal BETA";
+}
+
+@section BeforeMain {
+    <nav class="app-primary-navigation" aria-label="primary">
```

Comment:
There's only two pages so I'm torn on whether it's worth it, but for a larger site I'd definitely want this in a separate component, or as part of the page layout file with the selected tab being decided by the view model.

Reply:
Absolutely agreed, I wasn't best pleased with it as I was writing it! I'm inclined to leave it as-is for now, since I don't think we expect any more pages to be added, but I do completely agree